app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = []
+    for person_data in people:
+        people_list.append(Person(person_data["name"], person_data["age"]))
+
+    for person_data in people:
+        person = Person.people[person_data["name"]]
+        if "wife" in person_data and person_data["wife"]:
+            person.wife = Person.people[person_data["wife"]]
+        elif "husband" in person_data and person_data["husband"]:
+            person.husband = Person.people[person_data["husband"]]
```

Comment:
Instead of checking for the presence of keys using `in` and accessing them directly, use the `dict.get()` method. This approach simplifies the code and avoids potential `KeyError` exceptions, making it cleaner and more concise .